### PR TITLE
fix #80183 blog.doist.com

### LINKED ISSUE #80183: blog.doist.com
### Issue URL (Social Widget)
 [https://blog.doist.com/virtual-networking/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fblog.doist.com%2Fvirtual-networking%2F)
### Comment
Username: `@rakleed`
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Hide your Referrer from third-parties (blog.doist.com), <br>Hide your User-Agent ()
DNS filtering: | server: `https://dns.cloudflare.com/dns-query`<br>
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Russian
Userscripts: | https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js

### PR DESCRIPTION
**Pull request by wout-codes**
#80183

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -38,6 +38,7 @@ donbosco-medien.de,giantbomb.com,strefainwestorow.pl,narcity.com,lingohut.com,mt
...
 !
+blog.doist.com##div[class$="toc-share"]
```
Comment (Sergey-Lyapin) [inline SocialFilter/sections/specific.txt:41]: `blog.doist.com##.db-actionbar__toc-share` is more reliable

Comment (Sergey-Lyapin) [reply SocialFilter/sections/specific.txt:41]: and please make rules for newsletters and social buttons (in articles)

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply SocialFilter/sections/specific.txt:41]: @Sergey-Lyapin You're someone who can reproduce everything. I can't reproduce anything else. I wonder how you do it.

Comment (Sergey-Lyapin) [reply SocialFilter/sections/specific.txt]: I don't know, probably some experience =) https://github.com/AdguardTeam/AdguardFilters/commit/8f6396bbdfbbde66dc083678916eaea48f5b3566
Screenshot 2021-04-12 at 13 01 50
Screenshot 2021-04-12 at 13 02 06
Screenshot 2021-04-12 at 13 02 48
Screenshot 2021-04-12 at 13 09 19

**Pull request merged**